SCPlayer/SCPlayer/Controllers/Cores/Home Screen/HomeViewController.swift

```diff
@@ -34,21 +37,52 @@ final class HomeViewController: UIViewController {
             $0.register(cellType: HomeListTrackCollectionViewCell.self)
         }
     }
+    
+    private func getDataFromAPI() {
+        let shared = APIServices()
```

Comment:
Tôi thấy ông áp dụng Singleton trong APIServices rồi mà sao lại tạo mới đối tượng kiểu này nhỉ

Reply:
Oke . Tôi sửa rồi ông ạ. Call luông không cần tạo biến

Reply:
ông thêm đoạn này vào trong APIServices nhé, chứ singleton mà vẫn tạo mới được đối tượng thế này là dở rồi
```suggestion
       private init() {}
```